uwuno/src/main/java/com/learning/uwuno/util/typeUtil.java

```diff
@@ -0,0 +1,8 @@
+package com.learning.uwuno.util;
+
+public final class typeUtil {
```

Comment:
Can this just be a static class, if that doesn't exist in java ignore me.

Reply:
Only nested class can be static